src/wechaty_puppet_hostie/puppet.py

```diff
@@ -420,8 +421,13 @@ async def message_forward(self, to_id: str, message_id: str):
             await self.message_send_mini_program(conversation_id=to_id, mini_program=mini_program)
         # TODO
         # elif payload.type == MessageType.MESSAGE_TYPE_EMOTICON:
-        # elif payload.type == MessageType.MESSAGE_TYPE_AUDIO:
+        elif payload.type == MessageType.MESSAGE_TYPE_AUDIO:
+            # TODO
+            raise WechatyPuppetGrpcError('Can not support audio message forward')
```

Comment:
I think this error should be `WechatyPuppetOperationError`, how do you think about it ?